support-frontend/assets/pages/paper-subscription-landing/paperSubscriptionLandingPage.tsx

```diff
@@ -61,6 +62,7 @@ function PaperLandingPage({
 
 	const [selectedTab, setSelectedTab] =
 		useState<PaperFulfilmentOptions>(fulfilment);
+	windowHistoryUpdate(selectedTab); // set tab in url hash param upon page open
```

Comment:
I think maybe this isn't necessary, since you'll get the default tab selected. What do you reckon? It feels _slightly_ odd to do this without the user interacting with the page.